src/stan-users-guide/measurement-error.Rmd

```diff
@@ -227,7 +227,7 @@ the studies being analyzed.
 Suppose the data in question arise from a total of $M$ studies
 providing paired binomial data for a treatment and control group.  For
 instance, the data might be post-surgical pain reduction under a treatment
-of ibuprofen @WarnThompsonSpiegelhalter:2002 or mortality after
+of Ibuprofen [@WarnThompsonSpiegelhalter:2002] or mortality after
```

Comment:
Lower case was right.  "ibruprofen" is generic.  See https://www.merriam-webster.com/dictionary/ibuprofen.

Reply:
Aaah, I really thought it was a registered trademark, i.e., a name :) You learn something new every day :)